privaterelay/templates/profile.html

```diff
@@ -117,7 +117,7 @@
                   <span class="input-error"></span>
                 </div>
 
-                <button title="Click to copy"
+                <button title="{% ftlmsg 'click-to-copy' %}"
```

Comment:
Looking at the merged strings form https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/6, this should actually be `profile-label-click-to-copy`

```suggestion
                <button title="{% ftlmsg 'profile-label-click-to-copy' %}"
```